src/docfx/lib/markdown/validation/ContentValidationContext.cs

```diff
@@ -0,0 +1,52 @@
+// Copyright (c) Microsoft. All rights reserved.
+// Licensed under the MIT license. See LICENSE file in the project root for full license information.
+
+using System;
+using Validations.DocFx.Adapter;
+
+namespace Microsoft.Docs.Build
+{
+    internal class ContentValidationContext : IValidationContext
+    {
+        private const string ValidationApiBaseUrl = "validationApiBaseUrl";
+        private const string DefaultValidationApiBaseUrl = "https://docs.microsoft.com";
+        private const string ValidationRequestTimeoutInSeconds = "validationRequestTimeoutInSeconds";
+        private const int DefaultValidationRequestTimeoutInSeconds = 10;
+        private const string ValidationMaxTries = "validationMaxTries";
+        private const int DefaultValidationMaxTries = 4;
```

Comment:
Why retry 4 times? Usually this number is 3

Reply:
same as v2

Reply:
In addition, all config parameters are same as v2.